### PR TITLE
fix#64 fix check right holder of code/ feat/#65  check the case is any…

### PR DESCRIPTION
**Pull request by danielAlvess**
This merge did:

- fix bug can not check right holder of code in case of collective decision
- check the case right holder is any community members

### CODE REVIEW

**Comment by ntpdung** [comment]
Thank you @danielAlvess , LGTM!

**Comment by duyphanz** [comment]
@JackVuong please review this!

File: src/CanCommunity.ts [hunk 1]
```diff
@@ -239,9 +239,14 @@ export class CanCommunity {
...
 
-    // TODO check the case that is_any_community_member
-    if (rightHolder.is_any_community_member || rightHolder.is_anyone) {
-      return true;
+    if (rightHolder.is_any_community_member) {
+      const communityMember = await this.query(TableNameEnum.V1_MEMBER, {
+        upper_bound: account,
+        lower_bound: account,
+      });
+      if (communityMember && communityMember.rows && communityMember.rows[0] && communityMember.rows[0].member === account) {
```
Comment (JackVuong) [inline src/CanCommunity.ts:247]: Have you ever try `communityMember?.rows?.[0]?.member ` for shorter?

Comment (JackVuong) [reply src/CanCommunity.ts:247]: https://github.com/tc39/proposal-optional-chaining

Comment (danielAlvess) [reply src/CanCommunity.ts:247]: Nice

**Comment by danielAlvess** [comment]
@ntpdung  I resolve discussion!. please help me merge!

**Pull request merged**